src/dom/DomEvent.DoubleTap.js

```diff
@@ -65,7 +99,7 @@ export function addDoubleTapListener(obj, handler) {
 		if (now - last <= delay) {
 			detail++;
 			if (detail === 2) {
-				handler(makeDblclick(e));
+				ev.target.dispatchEvent(makeDblclick(ev));

```

Comment:
Are you sure that this results in the same?

Reply:
It's not "the same". It's... better. Now the event handlers do receive an instance of `MouseEvent` or `PointerEvent` instead of an empty object with copied properties. So the propagation & cancellation logic is the browser's native one. In other words: synthetic `dblclick` events will propagate through the DOM.

AFAIK this couldn't be done before, as browsers didn't have consistent behaviour for `PointerEvent`s, and/or different ways of instantiating synthetic events.

Reply:
Okay yeah I understand now. Is much more cleaner now!